plans/fix-1889-launcher-origin.md

```
@@ -0,0 +1,193 @@
+git で確定した変更点:
+
+| version | ブラウザに渡す URL |
+|---|---|
+| 4.10.1 | `bin/mulmoterminal.js:352` … `const url = \`http://localhost:${port}\`;` |
+| 4.11.0 | `launcherUrl(reachHost, port)` → 既定 bind `127.0.0.1` なので `http://127.0.0.1:<port>` |
+
+`launcherUrl` は PR #1877（issue #1876）で入った。
+
+## これは #1876 の修正ではない
+
+#1876 の報告された不具合は **二重起動ガードが既定構成で発火しない** こと ——
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)